### PR TITLE
Kanvi/remove files

### PR DESCRIPTION
**Pull request by kanvi-nervana**
ngtf_graph_viewer.py can now be removed because we can use the script from tensorflow import_pb_to_tensorboard.py + our tf2ngraph to do the same thing. So all that code is not required. I updated the README with the instructions on how to use it.
And the tools/run_tf2ngraph_model.py was uploaded accidentally with a PR for the aot change. It was not reviewed and is not required.

### CODE REVIEW

File: diagnostics/README.md [hunk 1]
```diff
@@ -68,40 +59,17 @@ For example usage, take a look at the ```model_test/verify_model.py``` in the di
...
-
-* `pbtxt` to TB: ```python ngtf_graph_viewer.py pbtxtfile.pbtxt ./vis```
-* `pbtxt` to dot: ```python ngtf_graph_viewer.py -v 0 pbtxtfile.pbtxt ./vis```
-* `pb` to TB: ```python ngtf_graph_viewer.py -b pbtxtfile.pb ./vis```
-* `pb` to dot: ```python ngtf_graph_viewer.py -b -v 0 pbtxtfile.pb ./vis```
-* `pbtxt` to TB (after prepending cluster information): ```python ngtf_graph_viewer.py -c nodemap.pkl pbtxtfile.pbtxt ./vis``` 
-
-
-### Some other use cases
+The [tf2ngraph] script can be used to convert tensorflow graph to a ngraph graph
```
Comment (sayantan-nervana) [inline diagnostics/README.md]: [tf2ngraph] 's link is not provided at the end of the file. So in the markdown rendering it does not show up as a link, but simply as `[tf2ngraph`

Comment (sayantan-nervana) [inline diagnostics/README.md]:
```suggestion
The [tf2ngraph] script can be used to convert tensorflow graph to an ngraph enabled graph
```

File: diagnostics/README.md [hunk 2]
```diff
@@ -68,40 +59,17 @@ For example usage, take a look at the ```model_test/verify_model.py``` in the di
...
+and dump it as a protobuf (`pb`).(https://github.com/tensorflow/ngraph-bridge/blob/master/tools/tf2ngraph.py)
+Tensorflows' `import_pb_tensorboard.py` script can then be used to view the
+ngraph graph on `tensorboard`.
```
Comment (sayantan-nervana) [inline diagnostics/README.md]:
```suggestion
dumped graph on `tensorboard`.
```

Comment (sayantan-nervana) [reply diagnostics/README.md]: tf2ngraph does not dump an ngraph graph. it dumps a TF graph, which is ngraph ready

File: diagnostics/README.md [hunk 3]
```diff
@@ -68,40 +59,17 @@ For example usage, take a look at the ```model_test/verify_model.py``` in the di
...
-* pbtxt to dot: ```from ngtf_graph_viewer import protobuf_to_dot; protobuf_to_dot(input_filename, dot_dir, input_binary=False)```
-* pb to TB: ```from ngtf_graph_viewer import protobuf_to_grouped_tensorboardt; protobuf_to_grouped_tensorboard(input_filename, dot_dir, input_binary=True)```
-* pb to dot: ```from ngtf_graph_viewer import protobuf_to_dot; protobuf_to_dot(input_filename, dot_dir, input_binary=True)```
-* pb to graphdef: ```from ngtf_graph_viewer import load_file; load_file(input_filename, input_binary=True)```
-* pbtxt to graphdef: ```from ngtf_graph_viewer import load_file; load_file(input_filename, input_binary=False)```
-* modify a graphdef's nodes names: ```from ngtf_graph_viewer import modify_node_names; modify_node_names(graph_def, node_map={"net1/node1":"e1/net1/node1"})```
+### Examples usage
 
+* python tf2ngraph --input_pbtxt test_axpy.pbtxt --output_nodes add --output_pb axpy_ngraph.pb --ng_backend CPU
+* Add `import ngraph_bridge` to `import_pb_tensorboard.py` script
```
Comment (sayantan-nervana) [inline diagnostics/README.md]:
```suggestion
* Add `import ngraph_bridge` to `import_pb_to_tensorboard.py` script
```

File: diagnostics/README.md [hunk 4]
```diff
@@ -68,40 +59,17 @@ For example usage, take a look at the ```model_test/verify_model.py``` in the di
...
+* python import_pb_to_tensorboard.py --model_dir axpy_ngraph.pb --log_dir test/
```
Comment (sayantan-nervana) [inline diagnostics/README.md]: Add TF link to `import_pb_to_tensorboard.py`

File: diagnostics/README.md (hunk 1 above)
Comment (kanvi-nervana) [reply diagnostics/README.md]: Should it be at the end of the file or end of line ?

Comment (sayantan-nervana) [reply diagnostics/README.md]: end of file. the .md file has a bunch of links at EOF. so if u add an entry there,  [tf2ngraph] will show up as a link

File: diagnostics/README.md [hunk 5]
```diff
@@ -44,15 +44,6 @@ To get a **full** dump use the following set of flags
 | `NGRAPH_TF_DUMP_DECLUSTERED_GRAPHS=1` | Dump graphs with final clusters assigned. Use this to view TF computation graph with colored nodes indicating clusters|
 |
```
Comment (kanvi-nervana) [inline diagnostics/README.md]:
```suggestion
```

File: diagnostics/README.md [hunk 6]
```diff
@@ -68,40 +59,19 @@ For example usage, take a look at the ```model_test/verify_model.py``` in the di
...
-* pb to TB: ```from ngtf_graph_viewer import protobuf_to_grouped_tensorboardt; protobuf_to_grouped_tensorboard(input_filename, dot_dir, input_binary=True)```
-* pb to dot: ```from ngtf_graph_viewer import protobuf_to_dot; protobuf_to_dot(input_filename, dot_dir, input_binary=True)```
-* pb to graphdef: ```from ngtf_graph_viewer import load_file; load_file(input_filename, input_binary=True)```
-* pbtxt to graphdef: ```from ngtf_graph_viewer import load_file; load_file(input_filename, input_binary=False)```
-* modify a graphdef's nodes names: ```from ngtf_graph_viewer import modify_node_names; modify_node_names(graph_def, node_map={"net1/node1":"e1/net1/node1"})```
+### Example usage
 
+* python tf2ngraph --input_pbtxt test_axpy.pbtxt --output_nodes add --output_pb axpy_ngraph.pb --ng_backend CPU
+* Add `import ngraph_bridge` to [import_pb_to_tensorboard.py] script
+* python import_pb_to_tensorboard.py --model_dir axpy_ngraph.pb --log_dir test/
```
Comment (sayantan-nervana) [inline diagnostics/README.md]:
```suggestion
* `python import_pb_to_tensorboard.py --model_dir axpy_ngraph.pb --log_dir test/`
```

File: diagnostics/README.md [hunk 7]
```diff
@@ -68,40 +59,19 @@ For example usage, take a look at the ```model_test/verify_model.py``` in the di
...
+* Launch Tensorboard by pointing it to the log directory : tensorboard --logdir=test/
```
Comment (sayantan-nervana) [inline diagnostics/README.md]:
```suggestion
* Launch Tensorboard by pointing it to the log directory : `tensorboard --logdir=test/`
```

File: diagnostics/README.md [hunk 8]
```diff
@@ -68,40 +58,19 @@ For example usage, take a look at the ```model_test/verify_model.py``` in the di
...
-* `pbtxt` to dot: ```python ngtf_graph_viewer.py -v 0 pbtxtfile.pbtxt ./vis```
-* `pb` to TB: ```python ngtf_graph_viewer.py -b pbtxtfile.pb ./vis```
-* `pb` to dot: ```python ngtf_graph_viewer.py -b -v 0 pbtxtfile.pb ./vis```
-* `pbtxt` to TB (after prepending cluster information): ```python ngtf_graph_viewer.py -c nodemap.pkl pbtxtfile.pbtxt ./vis``` 
-
-
-### Some other use cases
+The [tf2ngraph] script can be used to convert tensorflow graph to an ngraph enabled graph
+and dump it as a protobuf (`pb`).
+Tensorflows' `import_pb_tensorboard.py` script can then be used to view the
```
Comment (sayantan-nervana) [inline diagnostics/README.md]:
```suggestion
Tensorflow's `import_pb_tensorboard.py` script can then be used to view the
```

File: diagnostics/README.md [hunk 9]
```diff
@@ -68,40 +58,19 @@ For example usage, take a look at the ```model_test/verify_model.py``` in the di
...
-
-* `pbtxt` to TB: ```python ngtf_graph_viewer.py pbtxtfile.pbtxt ./vis```
-* `pbtxt` to dot: ```python ngtf_graph_viewer.py -v 0 pbtxtfile.pbtxt ./vis```
-* `pb` to TB: ```python ngtf_graph_viewer.py -b pbtxtfile.pb ./vis```
-* `pb` to dot: ```python ngtf_graph_viewer.py -b -v 0 pbtxtfile.pb ./vis```
-* `pbtxt` to TB (after prepending cluster information): ```python ngtf_graph_viewer.py -c nodemap.pkl pbtxtfile.pbtxt ./vis``` 
-
-
-### Some other use cases
+The [tf2ngraph] script can be used to convert tensorflow graph to an ngraph enabled graph
```
Comment (avijit-nervana) [inline diagnostics/README.md]: This is not very user friendly. Explain the use case in more detailed and provide better example.

**Pull request merged**